Unit-11/main.js

```diff
@@ -0,0 +1,97 @@
+// exercise-1
+const createElementMethodOne = (char,number) => {
+  const newArray = []
+  for(let i = 0;  i < number; i++) {
+    newArray.push(char)
+  }
+  return newArray
+}
+// exercise-1
+const createElementMethodTwo = (char,number) => {
+  const newArray = []
+  let i=0
+  while(i < number) {
+    newArray.push(char)
+    i+=1
+  }
+  return newArray
+}
+
+// exercise-2
+const createArrayReverse = (array = []) => {
+  const reverseArray = []
+  for(let i=array.length-1; i >= 0; i--) {
+    reverseArray.push(array[i])
+  }
+  return reverseArray
+}
+// exercise-3
+const deleteArray = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(typeof(array[i]) == 'number' && array[i] !=0) {
+      trueValues.push(array[i])
+    }
+  }
+  return trueValues
+}
+
+// exercise-4
+const createObject = (array = []) => {
+  const newObject = {}
+  array.forEach(item => {
+    newObject[item[0]] = item[1]
+  })
+  return newObject
+}
+// exercise-5
```

Comment:
2 khối khác nhau thì tách nhau ra
```suggestion
}

// Exercise-5
```